pkg/event/event_suite_test.go

```diff
@@ -0,0 +1,140 @@
+package event_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	k8sV1 "k8s.io/api/core/v1"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/event"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestRegistryMethods(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Event Registry suite")
+}
+
+type TestEvent struct {
+	Timestamp            int64
+	Name                 string
+	HandlerName          string
+	HandlerNetworkPlugin string
```

Comment:
These are unused.

Reply:
HandlerNetworkPlugin is used. I'll check the others.

Reply:
oh, right on the event it's not used.